scrabble.js

```diff
@@ -1,14 +1,149 @@
 
 const Scrabble = {
   score(word) {
+    let totalScore = 0;
+    if (typeof word !== 'string') {
+      throw "Invalid input, please enter a word!";
+    }
```

Comment:
Instead of throwing a string, you should throw an instance of `Error`. That gives you access to stack trace information. Something like this:

```javascript
if (condition) {
  throw new Error(`Invalid word ${word}`);
}
```